src/main/java/io/github/pmckeown/dependencytrack/finding/report/FindingsReport.java

```diff
@@ -12,13 +13,15 @@
 import java.util.stream.Collectors;
 
 @XmlRootElement(name = "findingsReport")
-@XmlType(propOrder = {"policyApplied", "policyBreached", "critical", "high", "medium", "low", "unassigned"})
+@XmlType(propOrder = {"policyApplied", "policyBreached", "policyViolations", "critical", "high", "medium", "low", "unassigned"})
 public class FindingsReport {
 
     private PolicyApplied policyApplied;
     private Boolean policyBreached;
     private List<Finding> findings;
 
+    private List<PolicyViolation> policyViolations;
```

Comment:
In line with my other comments, this would be better in a PolicyViolationsReport class